modules/administration-guide/partials/assembly_backing-up-a-che-instance-to-a-rest-backup-server-by-custom-resources.adoc

```diff
@@ -0,0 +1,17 @@
+:parent-context-of-backing-up-a-{prod-id-short}-instance-to-a-rest-backup-server-by-custom-resources: {context}
+
+[id="backing-up-a-{prod-id-short}-instance-to-a-rest-backup-server-by-custom-resources_{context}"]
```

Comment:
I have questions about using attributes like {prod-id-short} in ids like this. I'll check with #forum-docs-review to see if this causes any issues.

Reply:
WARNING: Keep in mind that changing the attribute value will break any xrefs that point to the hard-coded form of this id (without using the attribute). In other words, all xrefs that point to this topic must also use the attribute and not its value. If, or rather WHEN, someone changes the attribute value, any hard-coded xrefs might cause the builds to fail AFTER the PR that contains the change has merged. I caused a similar failure recently. Therefore, I recommend avoiding attribute values in IDs like this. (If you like, we can test this failure scenario.)

Reply:
@themr0c, FYI. Let's discuss this in https://issues.redhat.com/browse/RHDEVDOCS-3385